src/main/java/com/nitnelave/CreeperHeal/command/CreeperCommandManager.java

```diff
@@ -305,7 +305,7 @@ private void integerCmd(CfgVal key, String[] args, String help, String unit,
      *            to all worlds.
      */
     private void
-        forceCmd(String[] args, @SuppressWarnings("unused") String msg, CommandSender sender,
+        forceCmd(String[] args, String msg, CommandSender sender,
```

Comment:
Same as above, I think eclipse suggested it. I have really no idea what it does, but everything seems to work after the edit.